documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -114,12 +125,13 @@ metadata:
   labels:
     strimzi.io/cluster: my-cluster
 spec:
+  mode: full
```

Comment:
@ppatierno -- Not sure we need to specify this here, but we're not using default goals

Reply:
AFAIU the mode is independent from the goals. You can combine them in any way you want?

Reply:
Jakub is right. Goals and modes are not related at all.